spec/lib/email/processor_spec.rb

```diff
@@ -193,4 +193,26 @@
       ))
     end
   end
+
+  describe 'email_reject_too_many_recipients' do
```

Comment:
Minor but I feel like the description here doesn't really describe what we're testing below. Perhaps something like 

```suggestion
  describe 'when group email recipients exceeds maximum_recipients_per_new_group_email site setting' do
```